athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/metadata/ListTablesResponse.java

```diff
@@ -47,11 +48,27 @@
      */
     @JsonCreator
     public ListTablesResponse(@JsonProperty("catalogName") String catalogName,
-            @JsonProperty("tables") Collection<TableName> tables)
+                              @JsonProperty("tables") Collection<TableName> tables)
+    {
+        this(catalogName, tables, null);
+    }
+
+    /**
+     * Constructs a new ListTablesResponse object.
+     *
+     * @param catalogName The catalog name that tables were listed for.
+     * @param tables The list of table names (they all must be lowercase).
+     * @param nextToken The pagination starting point for the next request (null indicates the end of the pagination).
+     */
+    @JsonCreator
+    public ListTablesResponse(@JsonProperty("catalogName") String catalogName,
+                              @JsonProperty("tables") Collection<TableName> tables,
+                              @JsonProperty("nextToken") String nextToken)
```

Comment:
same comment, dual constructor makes me think we didn't do the compatibility work

Reply:
We are still using `version 2` of the SDK. Removing the original constructor will be the opposite of backwards compatibility. It would mean that many many files will have to change, not just in the SDK, but for every single connector as well as the control plane.

Reply:
Removed original constructor and tested all scenarios using a local stack.